lib/node_modules/@stdlib/blas/ext/base/sapxsumors/lib/ndarray.js

```diff
@@ -26,40 +26,39 @@ var float64ToFloat32 = require( '@stdlib/number/float64/base/to-float32' );
 // MAIN //
 
 /**
-* Adds a constant to each single-precision floating-point strided array element and computes the sum using ordinary recursive summation.
+* Adds a scalar constant to each single-precision floating-point strided array element and computes the sum using ordinary recursive summation.
 *
 * @param {PositiveInteger} N - number of indexed elements
-* @param {number} alpha - constant
+* @param {number} alpha - scalar constant
 * @param {Float32Array} x - input array
-* @param {integer} stride - stride length
-* @param {NonNegativeInteger} offset - starting index
+* @param {integer} strideX - stride length
+* @param {NonNegativeInteger} offsetX - starting index
 * @returns {number} sum
 *
 * @example
 * var Float32Array = require( '@stdlib/array/float32' );
-* var floor = require( '@stdlib/math/base/special/floor' );
 *
 * var x = new Float32Array( [ 2.0, 1.0, 2.0, -2.0, -2.0, 2.0, 3.0, 4.0 ] );
 *
 * var v = sapxsumors( 4, 5.0, x, 2, 1 );
 * // returns 25.0
 */
-function sapxsumors( N, alpha, x, stride, offset ) {
+function sapxsumors( N, alpha, x, strideX, offsetX ) {
 	var sum;
 	var ix;
 	var i;
 
 	if ( N <= 0 ) {
 		return 0.0;
 	}
-	if ( N === 1 || stride === 0 ) {
-		return float64ToFloat32( alpha + x[ 0 ] );
+	ix = offsetX;
+	if ( strideX === 0 ) {
+		return float64ToFloat32( N * ( alpha + x[ ix ] ) );
```

Comment:
```suggestion
		return float64ToFloat32( N * float64ToFloat32( alpha + x[ ix ] ) );
```

In order to emulate single-precision floating-point arithmetic, all intermediate computations need to be wrapped. This ensures 1:1 parity with C.